src/accessibility/KeyboardShortcuts.ts

```diff
@@ -276,8 +278,22 @@ export const shortcuts: Record<Categories, IShortcut[]> = {
             description: _td("Cancel autocomplete"),
         },
     ],
+
+    // Intentionally left blank, as bindings should be only added when the feature is enabled
+    [Categories.LABS]: [],
 };
 
+// Add this short only if labs are enabled, as the feature is only exposed there.
+if (SdkConfig.get()['showLabsSettings']) {
+    shortcuts[Categories.LABS].push({
+        keybinds: [{
+            modifiers: [CMD_OR_CTRL, Modifiers.SHIFT],
+            key: Key.H,
+        }],
+        description: _td("Toggle visibility of hidden events"),
+    });
+}
```

Comment:
This should probably use `registerShortcut()` and be somewhere else, I think.

Reply:
What would be the good place for that? I can't find any calls to that function anywhere within code.

Reply:
From my poking around, `SdkConfig` is not initialized when `shortcuts` are being defined, so the key shortcuts will have to be defined somewhere else, although I can't find any use of the `registerShortcut` function, so I'm left guessing.

Reply:
Perhaps a call from https://github.com/vector-im/element-web/blob/develop/src/vector/index.ts to a function in https://github.com/vector-im/element-web/blob/develop/src/vector/init.tsx at the correct time would work?

Though someone more experienced might have a better idea

Reply:
Ah, registerShortcut is used from the element-web in the electron-specific code: https://github.com/vector-im/element-web/blob/develop/src/vector/platform/ElectronPlatform.tsx#L290

And yep, Simon's suggestion seems like a sensible place (`registerShortcuts` maybe).

Reply:
You could also presumably add the shortcut anyway and just do nothing if labs aren't shown.

Reply:
@dbkr actually, that might be a good idea. The list of categories of shortcuts is defined in the shortcuts tab, and I managed to add a check there to append the Labs section to the list. Having the shortcut defined unconditionally in one place as everything else might make things easier to follow.